tasks/facial_landmark_detection/train.py

```diff
@@ -148,6 +150,22 @@ def load_data(traindir, args):
     return dataset, dataset_test, train_sampler, test_sampler
 
 
+def setup_qat(model: nn.Module) -> None:
+    assert model is not None
+
+    _replace_relu(model)
+
+    model.eval()
+
+    # Modify quantization engine as appropriate for the target platform
+    model.setup_qconfig('fbgemm')
+    model.classifier.qconfig = None
```

Comment:
It seems the default for classifier.qconfig is already None. Should we make this clear? See the comment in readme as well.